CORE/elements/_core_element.rb

```diff
@@ -32,8 +32,9 @@ def initialize(name, world, options)
     raise "ELEMENT [#{name}] must have a locator!/nValid locators are: #{@@locator_options}" unless has_locator
 
     @world = world
-    @options = {:active => true}.merge(options)
+    @options = {:active => true, :state => 'enabled'}.merge(options)
```

Comment:
Would it make sense to use a symbol here instead of a string?

Reply:
Good point. I still manage to think in other languages sometimes and am not used to symbols. Will be updating.